charts/astronomer/templates/houston/houston-configmap.yaml

```diff
@@ -590,7 +590,7 @@ data:
           elasticsearch:
             enabled: true
             connection:
-              host: {{ printf "%s-external-es-proxy.%s" .Release.Name .Release.Namespace }}
+              host: elasticsearch.{{ .Values.global.plane.domainSuffix }}.{{ .Values.global.baseDomain }}
```

Comment:
This should use the `elasticsearch.url` template defined below, right?

Reply:
this might no longer be valid since the metadata url will expose these informations

Reply:
this is for airflow elasticsearch so technically no public url is needed here we will need only the local url, existing format is correct in this case. since all airflow deployment will connect